### PR TITLE
UX Design Task Submission

### PR DESCRIPTION
**Pull request by kolawolejohn**
- added process flow design
- added user story
-  added mock-up for my uxDesign

### CODE REVIEW

**Review by goke-epapa — COMMENTED** [review]
Please add a description to this PR

You can refer to the guide here: https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19#pull-request

Also please update the title of the PR to reflect the task which is `UX Design`

File: README.md [hunk 1]
```diff
@@ -36,3 +36,12 @@ when creating a pull request, you need to inform us of what you have done or cha
 # Keeping your Fork in Sync
 
 As more content is added to this repository and more people contribute, your fork will fall out of date. This can sometimes lead to tricky situations with your pull requests. It's recommended to occasionally sync your fork with this [repository](https://help.github.com/articles/syncing-a-fork/).
+
+# Task so far
+
+I created a folder with my full name in the submissions folder, I created a subfolder named uxDesign in it and I added my Dufuna uxDesign tasks which are
+- User Story
+- Process flow
+- uxDesign
```
Comment (ToluAkin) [inline README.md]: @kolawolejohn Please don't edit the README.

File: README.md [hunk 2]
```diff
@@ -36,3 +36,6 @@ when creating a pull request, you need to inform us of what you have done or cha
 # Keeping your Fork in Sync
 
 As more content is added to this repository and more people contribute, your fork will fall out of date. This can sometimes lead to tricky situations with your pull requests. It's recommended to occasionally sync your fork with this [repository](https://help.github.com/articles/syncing-a-fork/).
+
```
Comment (yemexx1) [inline README.md]: Could you please revert this change? We shouldn't have blank lines there

**Review by yemexx1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (otaruMendez) [inline README.md]: Welldone so far @kolawolejohn, can you remove these 3 spaces still showing here. The README.md file shouldn't show in your PR at all.

Comment (kolawolejohn) [reply README.md]: I have done that. Thanks

Comment (kolawolejohn) [reply README.md]: I have done that sir.

Comment (otaruMendez) [reply README.md]: > I have done that. Thanks
Okay, I can still see the ReadMe in the files changed session though, can you try to make sure it doesn't show at all?
